var/spack/repos/builtin/packages/lbann/package.py

```diff
@@ -112,6 +112,8 @@ class Lbann(CMakePackage):
     depends_on('py-onnx@1.3.0:', type='run', when='@:0.90,0.99:')
     depends_on('py-pandas@0.24.1:', type='run', when='@:0.90,0.99:')
     depends_on('py-texttable@1.4.0:', type='run', when='@:0.90,0.99:')
+    depends_on('py-pytest', type='run', when='@:0.90,0.99:')
```

Comment:
Usually pytest is a `type='test'` dependency. How is it used at run-time in this package?